### PR TITLE
297 saveoutputtotextfile ddt wide rollout

### PR DESCRIPTION
**Pull request by CAfitech**
We have updated the code of all tools and attack vectors to incorporate saveoutputtotectfil_v2.

### CODE REVIEW

File: .yarn/install-state.gz
Comment (zkkok) [inline .yarn/install-state.gz:1]: Please do not commit this file

File: .yarnrc.yml
Comment (zkkok) [inline .yarnrc.yml:1]: Please do not commit this file

File: yarn.lock
Comment (zkkok) [inline yarn.lock:1]: Please do not commit this file

**Review by zkkok — CHANGES_REQUESTED** [review]
Hi @hhenryDeakin @CAfitech,
Appreciate the great work both of you have produced. However, it looks like your code have generated some conflicts. This might be due to some commits you have done one some of the yarn files. The rule of thumb for committing the files is to only commit those you have been working in. Files like yarnrc.yml, yarn.lock are automated processes that does not need to be committed.

Additionally here's a guide on writing better commit messages and pull requests:
https://www.freecodecamp.org/news/how-to-write-a-pull-request-description/
https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/

File: yarn.lock
Comment (CAfitech) [reply yarn.lock:1]: Thanks, I have made the adequate changes.

File: .yarnrc.yml
Comment (CAfitech) [reply .yarnrc.yml:1]: Thanks, I have made the adequate changes.

File: .yarn/install-state.gz
Comment (CAfitech) [reply .yarn/install-state.gz:1]: Thanks, I have made the adequate changes.

File: yarn.lock
Comment (zkkok) [reply yarn.lock:1]: Hi @CAfitech, you have not removed the yarn.lock file

**Comment by bradleybogg** [comment]
Hi Team, please make the requested changes. When ready then create the PR. Thank you!

**Pull request closed without merging**